ignite/pkg/events/events.go

```diff
@@ -7,6 +7,10 @@ import "fmt"
 // ProgressIndication enumerates possible states of progress indication for an Event
 type ProgressIndication uint8
 
+const (
+	GroupError = "error"
+)
+
 const (
```

Comment:
```suggestion
const (
	GroupError = "error"

```

Reply:
I think it is not possible because of the iota 🤔

GroupError on top would cause the iota to start from 1 and moving it down the group would look weird being grouped with the iota initialised constants. What do you think @aljo242?

Reply:
ok - totally fine.  Wasn't a big deal anyways!